.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: ruby
```

Comment:
will it pick the correct version from .ruby-version, or does it need the:
`rvm:`
`- 2.3.1`
bit?